crates/ef-testing-proc-macro/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "ef-testing-proc-macro"
```

Comment:
this name is not really expressive imho,
i don't think we should name a crate after what its main goal/ its main attribute

Reply:
I followed the naming from cairo corelib for proc-macros: https://github.com/starkware-libs/cairo/tree/main/crates/cairo-lang-proc-macros

Reply:
mhmm ok but here we'll only ever have one, no?

Reply:
okay lgtm

Reply:
cf also https://doc.rust-lang.org/book/ch19-06-macros.html

> When creating procedural macros, the definitions must reside in their own crate with a special crate type. This is for complex technical reasons that we hope to eliminate in the future.